src/users/getters.ts

```diff
@@ -18,17 +19,11 @@ export const getUserProfileName = ($userProfile: UserProfile) => {
 const PSEUD_SUFFIX = ", ";
 export const getUserProfilePseuds = ($userProfile: UserProfile) => {
   const pseuds = $userProfile("dd.pseuds a");
-  const pseudsArray: string[] = [];
-
-  if (pseuds.length !== 0) {
-    pseuds.each((i, element) => {
-      const url = element.attribs.href;
-      const [, username, pseud] = url.match(/users\/(.+)\/pseuds\/(.+)/)!;
-
-      pseudsArray.push(decodeURI(pseud));
-    });
-  }
-  return pseudsArray.join(PSEUD_SUFFIX);
+  return pseuds.length !== 0 
+    ? pseuds.map((_, element) => decodeURI(element.attribs.href.match(/users\/(.+)\/pseuds\/(.+)/)![2]))
+    .get()
```

Comment:
is there a reason why you changed this? it makes it harder to understand what's going on.

I would 
1) if `pseuds` is empty, immediately return `""`
2) if not, do `pseuds.map` like you're doing now
3) keep the inner function (`const url = element.attribs.href;` etc) as is, but return instead of adding to the array.